front-end/src/doser/main.jsx

```diff
@@ -24,10 +22,6 @@ class doser extends React.Component {
     this.calibrateDoser = this.calibrateDoser.bind(this)
   }
 
-  dismissModal () {
```

Comment:
CalibratonModal closes properly without this? I think we have a bug that I have to fix on master,

Reply:
Yes. Moreover I would suggest that the "Run" button change to "Stop" after being activated and that it actually stops the doser. The dialog ignores the user's intention to stop the calibration procedure at will and continues until the end of the entered period. Pressing "Done" only dismisses the dialog but not the dosing so this should be fixed as well. For a regular person, "Done" means **Done** so perhaps a better term is warranted.